local-volume/helm/provisioner/templates/provisioner-service-account.yaml

```diff
@@ -1,6 +1,7 @@
 {{- if .Values.common.rbac }}
 apiVersion: v1
 kind: ServiceAccount
-metadata:  
+metadata:
   name: {{ .Values.daemonset.serviceAccount }}
+  namespace: {{ .Values.common.namespace }}
```

Comment:
I think service account could reside in a different namespace than provisioner's running pod. Instead of using namespace from `common`, for greater flexibility service account should have its own name space variable?

Reply:
it seems it was me, who was missing that point :)
/lgtm